### ISSUE TITLE
Maintenance: "newDagreD3" switch in the flowchart renderer is no longer needed and can be removed

### ISSUE THREAD

**Issue by GDFaber** [mermaid-js/mermaid#1182]
In order to move safely to the dagre-d3 branch, the flowchart renderer was given the possibility to switch between old and new version for testing purposes. This switch can now be removed.

This issue is a reminder for removing the code for the old dagre-d3 variant in the flowchart renderer. I'll PR the change after #1176 has been successfully merged because maybe there would be merge conflicts.

**Comment by GDFaber** [comment]
done
